meadow-client/src/Editor.purs

```diff
@@ -146,6 +148,7 @@ import Data.String as String
 import LocalStorage as LocalStorage
 import StaticData as StaticData
 
+isNotSuccess :: forall r e a. RemoteData r (Either e a) -> Boolean
 isNotSuccess (Success (Right _)) = false
```

Comment:
I believe you can just define this as `isNotSuccess = not isSuccess`. (Because `not` works on functions that return `Boolean`.)